### PR TITLE
NAT info modules gateway, snat, dnat

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: plugins/modules/nat_snat_info.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        id_filter = self.params['id']
+        nat_gateway_id_filter = self.params['nat_gateway_id']
+        network_id_filter = self.params['network_id']
+        source_type_filter = self.params['source_type']
+        status_filter = self.params['status']
+        project_id_filter = self.params['project_id']
+
+        data = []
+
+        for raw in self.conn.nat.snat_rules():
```
Comment (gtema) [inline plugins/modules/nat_snat_info.py:173]: and here as well

File: plugins/modules/nat_snat_info.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                continue
+            if (project_id_filter and raw.project_id != project_id_filter):
+                continue
+            dt = raw.to_dict()
+            dt.pop('location')
+            data.append(dt)
+
+        self.exit(
+            changed=False,
+            snat_list=data
```
Comment (gtema) [inline plugins/modules/nat_snat_info.py:208]: nat_snat_rules please

File: plugins/modules/rds_flavor_info.py [hunk 3]
```diff
@@ -57,14 +57,14 @@
         name:
             description: Name (version) of the datastore.
             type: str
-            sample: "10.0"
```
Comment (gtema) [inline plugins/modules/rds_flavor_info.py:60]: doesn't belong into this PR. In future please create separate PR for unrelated issues

File: tests/sanity/ignore-2.9.txt [hunk 4]
```diff
@@ -4,7 +4,9 @@ plugins/modules/cce_cluster.py validate-modules:missing-gplv3-license
 plugins/modules/cce_cluster_cert_info.py validate-modules:missing-gplv3-license
 plugins/modules/cce_cluster_info.py validate-modules:missing-gplv3-license
 plugins/modules/loadbalancer.py validate-modules:missing-gplv3-license
-plugins/modules/loadbalancer_info.py validate-modules:missing-gplv3-license
```
Comment (gtema) [inline tests/sanity/ignore-2.9.txt:7]: and why you drop loadbalancer_info?

**Review by gtema — CHANGES_REQUESTED** [review]

**Comment by tischrei** [comment]
Reworked everthing.

File: plugins/modules/nat_gateway_info.py [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    description:
+      - ID of the router where the NAT gateway is attached.
+    type: str
+  spec:
+    description:
+      - Specifies the type of the NAT gateway. 1 (small 10.000 connections),
+        2 (medium 50.000 connections), 3 (large 200.000 connections), 4
+        (extra-large 1.000.000 connections)
+    type: str
+    choices: ["1", "2", "3", "4"]
```
[the remark this answers is not in the document]
Comment (tischrei) [reply plugins/modules/nat_gateway_info.py:57]: it is expected as string and not as int

**Comment by tischrei** [comment]
recheck

**Review by gtema — APPROVED** [review]

**Pull request merged**